### ISSUE TITLE
Call to a member function getMedia() on array, on resource detail

### ISSUE THREAD

**Issue by nikita-nikolaevich** [whitecube/nova-flexible-content#153]
Using ebess/advanced-nova-media-library+whitecube/nova-flexible-content via Layouts. Everything works perfectly but after update v0.1.12 => v0.1.15 on resource detail page(after saving) appears error "Call to a member function getMedia() on array"

**Comment by jornwildenbeest** [comment]
Same problem here.

**Comment by zareismail** [comment]
Hi,
I resolved it.

add the  following into custom layout:

```  
    /**
     * Resolve fields for display using given attributes.
     *
     * @param array $attributes
     * @return array
     */
    public function resolveForDisplay(array $attributes = [])
    {
        $this->fields->each(function ($field) use ($attributes) {
            $field->resolveForDisplay(
                is_subclass_of($field, \Ebess\AdvancedNovaMediaLibrary\Fields\Media::class) 
                           ? $this->getMediaModel() : $attributes
            );
        });

        return $this->getResolvedValue();
    }

```

**Comment by Nyratas** [comment]
Hi @zareismail,

Thanks for the update. Could you maybe integrate this in a PR for the existing [HasMediaLibrary](https://github.com/whitecube/nova-flexible-content/blob/master/src/Concerns/HasMediaLibrary.php) trait ? Thank you !

**Comment by zareismail** [comment]
> Hi @zareismail,
> 
> Thanks for the update. Could you maybe integrate this in a PR for the existing [HasMediaLibrary](https://github.com/whitecube/nova-flexible-content/blob/master/src/Concerns/HasMediaLibrary.php) trait ? Thank you !


It's done

**Comment by voidgraphics** [comment]
Fixed, will be included in the next release

**Comment by voidgraphics** [comment]
Included in [v0.2.1](https://github.com/whitecube/nova-flexible-content/releases/tag/v0.2.1)
